charm_test.go

```diff
@@ -219,3 +225,88 @@ func cloneDir(c *gc.C, path string) string {
 	c.Assert(err, gc.IsNil)
 	return newPath
 }
+
+func (s *CharmSuite) assertVersionFile(c *gc.C, dir string, execName string, args []string) {
```

Comment:
Can we rename dir to charmDir so it's clear what dir it is
Actually, we are always passing in the same charm path for "dummy"  so no need to make this a param. What I was meaning in my original comment was to pass in the vcs dir, ".git" etc. But since it seems we can construct that always from the execName, then there's no need to pass it in at all. So this dir param can go.

Reply:
done